qa/suites/deepsea/tier4/upgrade/3-test-phase.yaml

```diff
@@ -16,13 +13,23 @@ tasks:
                         - 'zypper --non-interactive --no-gpg-checks dup --auto-agree-with-licenses --replacefiles'
         - deepsea.reboot:
                 client.salt_master:
+        - deepsea:
+                upgrade_install: package
+        - exec:
+                client.salt_master:
+                        - 'deepsea --version || true'
+                        - 'salt-run deepsea.version || true'
         - deepsea.repository:
                 all:
         - exec:
                 all:
                         - 'zypper --non-interactive --no-gpg-checks rm gvfs-backends sle-sdk-release-POOL || true'
         - salt.command:
                 command: 'state.apply ceph.updates.salt'
+        - salt.command:
```

Comment:
salt is only partly installed (due to unresolved vendor changes. weirdly enough this only happens on 2/4 nodes repeatedly)

```
Problem: salt-api-2016.11.4-48.9.33.x86_64 requires salt = 2016.11.4-48.9.33, but this requirement cannot be provided
  deleted providers: salt-2016.11.4-48.9.33.x86_64
 Solution 1: Following actions will be done:
  install salt-api-2019.2.0rc1-5.2.x86_64 (with vendor change)
    obs://build.suse.de/Devel:Storage:5.0  -->  SUSE LLC <https://www.suse.com/>
  install salt-master-2019.2.0rc1-5.2.x86_64 (with vendor change)
    obs://build.suse.de/Devel:Storage:5.0  -->  SUSE LLC <https://www.suse.com/>
 Solution 2: do not install salt-2019.2.0rc1-5.2.x86_64
 Solution 3: break salt-api-2016.11.4-48.9.33.x86_64 by ignoring some of its dependencies

Choose from above solutions by number or cancel [1/2/3/c] (c): y
Invalid answer 'y'. [1/2/3/c] (c): 1
Resolving dependencies...
Resolving package dependencies...
```